scripts/test_install_aktualizr_and_update.sh

```diff
@@ -2,28 +2,26 @@
 
 set -exuo pipefail
 
-TEST_INSTALL_DESTDIR=${TEST_INSTALL_DESTDIR:-/persistent}
+#TEST_INSTALL_DESTDIR=${TEST_INSTALL_DESTDIR:-/persistent}
+TEST_INSTALL_DESTDIR=$(mktemp -d)/aktualizr
 
-"$TEST_INSTALL_DESTDIR/selfupdate_server.py" 8000 "$TEST_INSTALL_DESTDIR"&
+./scripts/selfupdate_server.py 8000 "$TEST_INSTALL_DESTDIR"&
 
-dpkg-deb -I "$TEST_INSTALL_DESTDIR"/aktualizr*.deb && dpkg -i "$TEST_INSTALL_DESTDIR"/aktualizr*.deb
-akt_version=$(aktualizr --version)
-(grep "$(cat "$TEST_INSTALL_DESTDIR"/aktualizr-version)" <<< "$akt_version") || (echo "$akt_version"; false)
+$1/src/aktualizr_primary/aktualizr --version
 
-uptane-generator generate --path "$TEST_INSTALL_DESTDIR/fake_root"
-uptane-generator image --path "$TEST_INSTALL_DESTDIR/fake_root" --targetname selfupdate_2.0 --filename "$TEST_INSTALL_DESTDIR/selfupdate_2.0" --hwid selfupdate-device
-uptane-generator addtarget --path "$TEST_INSTALL_DESTDIR/fake_root" --targetname selfupdate_2.0 --hwid selfupdate-device --serial 723f79763eda1c753ce565c16862c79acdde32eb922d6662f088083c51ffde66
-uptane-generator signtargets --path "$TEST_INSTALL_DESTDIR/fake_root"
+$1/src/uptane_generator/uptane-generator generate --path "$TEST_INSTALL_DESTDIR/fake_root"
+$1/src/uptane_generator/uptane-generator image --path "$TEST_INSTALL_DESTDIR/fake_root" --targetname selfupdate_2.0 --filename "$TEST_INSTALL_DESTDIR/selfupdate_2.0" --hwid selfupdate-device
+$1/src/uptane_generator/uptane-generator addtarget --path "$TEST_INSTALL_DESTDIR/fake_root" --targetname selfupdate_2.0 --hwid selfupdate-device --serial 723f79763eda1c753ce565c16862c79acdde32eb922d6662f088083c51ffde66
+$1/src/uptane_generator/uptane-generator signtargets --path "$TEST_INSTALL_DESTDIR/fake_root"
 
 TEMP_DIR=$(mktemp -d)
 mkdir -m 700 -p "$TEMP_DIR/import"
 cp "$TEST_INSTALL_DESTDIR"/prov_selfupdate/* "$TEMP_DIR/import"
 echo -e "[storage]\\npath = \"$TEMP_DIR\"\\n[import]\\nbase_path = \"$TEMP_DIR/import\"" > "$TEMP_DIR/conf.toml"
 aktualizr -c "$TEST_INSTALL_DESTDIR"/selfupdate.toml -c "$TEMP_DIR/conf.toml" once
 
-# check that the version was updated
-akt_version=$(aktualizr --version)
-(grep 2.0-selfupdate <<< "$akt_version") || (echo "$akt_version"; false)
+# check the updated file appeared in the installation directory
+#folder="$TEST_INSTALL_DESTDIR/"
+#if [ ! -d "$folder"]; then
+#fi
```

Comment:
The downloaded and "installed" file should show up something in the storage directory, and we should verify that it is there and maybe that the sha256sum matches our expectations.